src/applications/vre/25-8832/containers/ConfirmationPage.jsx

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import moment from 'moment';
+import { connect } from 'react-redux';
+
+import scrollToTop from 'platform/utilities/ui/scrollToTop';
+import { focusElement } from 'platform/utilities/ui';
+import Telephone, {
+  CONTACTS,
+} from '@department-of-veterans-affairs/component-library/Telephone';
+
+import ServiceProvidersText, {
+  ServiceProvidersTextCreateAcct,
+} from 'platform/user/authentication/components/ServiceProvidersText';
+
+export class ConfirmationPage extends React.Component {
+  componentDidMount() {
+    focusElement('#thank-you-message');
+    scrollToTop('topScrollElement');
+  }
+
+  render() {
+    const { submission, data } = this.props.form;
+    const { isLoggedIn, fullName } = this.props;
+    const { response } = submission;
+    const name = isLoggedIn ? fullName : data.fullName;
+
+    return (
+      <div>
+        <p>
+          Equal to VA Form 28-8832 (Education/Vocational Counseling Application)
```

Comment:
Update

Reply:
Actually, this is correct. There is a specific note on the page indicating the forms are equivalent. Since titles and breadcrumbs indicate this is form 25-8832, this notes the form equivalence.